tests/reftests/run.ml

```diff
@@ -165,10 +165,7 @@ let run_test t ?vars ~opam =
   let opamroot0 = Filename.concat (Sys.getcwd ()) ("root-"^t.repo_hash) in
   with_temp_dir @@ fun dir ->
   let opamroot = Filename.concat dir "OPAM" in
-  if OpamStd.Sys.(os () <> Win32) then
-    command "cp -a %s %s" opamroot0 opamroot
-  else
-    command "robocopy /e %s %s" opamroot0 opamroot;
+  command "cp -a %s %s" opamroot0 opamroot;
```

Comment:
What does this fix? This has been worked on further in #4504 - cf. https://github.com/ocaml/opam/pull/4504/commits/06222ee4b1d1bd9a35445b029691eca4633cd498 and https://github.com/ocaml/opam/pull/4504/commits/769c3b6b5b1cee1cad242b2443a6739d84705e6c

Reply:
nevermind, sorry about that, I thought this would solver this issue: https://ci.appveyor.com/project/AltGr/opam/builds/37239388/job/0dvq95vl3qpneadn and thought i could quickly test, however in #4531 we can see that this did not help.
Could someone revert my commit if this branch is still being worked on? I can't force-push on this branch for some reason.